SHAPETEST/StartShapeTestLosProbe.md

```diff
@@ -23,6 +23,7 @@ enum TraceFlags
   IntersectWater = 32,
   Unknown = 128,
   IntersectFoliage = 256,
+  Unkown = 511, (used most)
```

Comment:
Unk**n**own is the correct spelling

Reply:
`511` is basically "everything" (`1+ 2 + 4 + 8 + 16 + 32 + 64 + 128 + 256`)

Reply:
> Unk**n**own is the correct spelling

Shit just noticed

Reply:
> `511` is basically "everything" (`1 + 2 + 4 + 8 + 16 + 32 + 64 + 128 + 256`), it's not a single flag

Alright, I'll make the change